umd.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'ky/umd' {
+	import ky from 'ky';
+
+	export default ky;
+	export * from 'ky';
```

Comment:
I'm not that familiar with export syntax, but are you sure this doesn't also export the default?

Reply:
Only `export * from 'ky';` doesn't work for my `typescript@3.1.6`